examples/vision/image_classification_from_scratch.py

```diff
@@ -328,6 +328,7 @@ def make_model(input_shape, num_classes):
 img = keras.preprocessing.image.load_img(
     "PetImages/Cat/6779.jpg", target_size=image_size
 )
+plt.imshow(img); # Display test image
```

Comment:
Remove `;` in all 3 files

Reply:
@suvigyajain0101 Are you planning to make this change? Let us know if you're still working on this. Otherwise we'll close the request. Thanks!

Reply:
Hi @pcoet,

Thanks for reaching out! I've updated the code as per comments from @fchollet and submitted a new PR (#1500)

Please let me know in case there are further changes required
Thanks!